app/tasks/research_tasks.py

```
@@ -691,8 +762,376 @@ def _generate_placeholder_report(job: dict, artifacts: list) -> str:
 """
+
+
+# ==============================================================================
+# Task 181: Report Generation Helpers
+# ==============================================================================
+
+
+async def _generate_report_with_executor(
+    job: Dict[str, Any],
+    artifacts: List[Dict[str, Any]],
+    tasks: List[Dict[str, Any]]
```

Ruff (0.14.13)

Unused function argument: `tasks`

(ARG001)